.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,34 @@
+name: build-and-push
+
+on:
+ release:
+   types:
+     - published
```

Comment:
Let's chage lines 3-6 temporarily to:

```yaml
on:
  pull_request:
  push:
    branches: [master]
```

so we can see it run here. We'll move it back once this performs as expected.